pkg/executor/join/hash_join_stats.go

```diff
@@ -21,9 +21,23 @@ import (
 	"sync/atomic"
 	"time"
 
+	"github.com/pingcap/tidb/pkg/util"
 	"github.com/pingcap/tidb/pkg/util/execdetails"
 )
 
+func convertBytesStatsToString(bytes []int64) string {
+	info := "["
```

Comment:
Better use bytes.NewBuffer instead of raw strings here.

Reply:
> Better use bytes.NewBuffer instead of raw strings here.

done